rfc021-platform-support-policy.md

```diff
@@ -33,7 +33,7 @@ Platform | Versions | Architectures | Package Format | Built on
 --- | --- | --- | --- | ---
 AIX | 6.1, 7.1 | ppc64 | bff | AIX 6.1
 CentOS | 5, 6, 7 | i386, x86_64 | rpm | RHEL 5
-Debian | 6, 7, 8 | i386, x86_64 | deb | Debian 6
+Debian | 7, 8 | i386, x86_64 | deb | Debian 6
```

Comment:
Indeed we is.   It came up recently because we started down the road of shaving that yak in EC2 for the new acceptance tests (testing chef-client against a pile of cookbooks automatically before we ship it) and @jkeiser was getting sucked into the yak hole and I pulled him out before he fell all the way in:

https://github.com/chef/chef/pull/4461